ui/components/phase-banner.jsx

```diff
@@ -1,11 +1,19 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import ReactMarkdown from 'react-markdown';
 
 class PhaseBanner extends React.Component {
 
   renderContent() {
     if (this.props.content) {
-      return this.props.content;
+      return (
+        <ReactMarkdown
+          renderers={{ root: Fragment, paragraph: ({ children }) => <span>{ children }</span> }}
+          linkTarget="_blank"
+        >
+          {this.props.content}
+        </ReactMarkdown>
+      );
```

Comment:
This feels like a massive dance just to avoid `this.props.content` being defined as jsx.